tests/functional/test_cluster_functions.py

```diff
@@ -9,443 +9,249 @@
-        steps.delete_all_data(client)
-
-    def test_shouldnt_create_service_w_id_eq_negative_number(self, client):
-        cluster = steps.create_cluster(client)
-        with allure.step('Try to create service with id as a negative number'):
+        with allure.step("Create a list of services in the cluster"):
+            for prototype in sdk_client_fs.service_prototype_list():
+                service = cluster.service_add(prototype_id=prototype.id)
+                expected.append(service._data)
+        with allure.step("Get a service list in cluster"):
+            actual = [x._data for x in cluster.service_list()]
+        with allure.step("Check expected and actual value"):
+            assert actual == expected
+
+    def test_should_not_create_service_w_id_eq_negative_number(
+        self, sdk_client_fs: ADCMClient, cluster: Cluster
+    ):
+        with allure.step("Try to create service with id as a negative number"):
             with pytest.raises(coreapi.exceptions.ErrorMessage) as e:
-                client.cluster.service.create(
-                    cluster_id=cluster['id'],
-                    prototype_id=(get_random_service(client)['id'] * -1))
-        with allure.step('Check error prototype doesn\'t exist'):
-            err.PROTOTYPE_NOT_FOUND.equal(e, 'prototype doesn\'t exist')
-        steps.delete_all_data(client)
-
-    def test_souldnt_create_service_w_id_as_string(self, client):
-        cluster = steps.create_cluster(client)
-        with allure.step('Try to create service with id as a negative number'):
+                prototype = sdk_client_fs.service_prototype()
+                _create_service(cluster, prototype.id * -1)
+        with allure.step("Check error prototype doesn't exist"):
+            err.PROTOTYPE_NOT_FOUND.equal(e, "prototype doesn't exist")
+
+    def test_should_not_create_service_w_id_as_string(self, cluster: Cluster):
+        with allure.step("Try to create service with id as a random string"):
             with pytest.raises(coreapi.exceptions.ErrorMessage) as e:
-                client.cluster.service.create(cluster_id=cluster['id'],
-                                              prototype_id=utils.random_string())
-        with allure.step('Check error about valid integer'):
-            assert e.value.error.title == '400 Bad Request'
-            assert e.value.error['prototype_id'][0] == 'A valid integer is required.'
-        steps.delete_all_data(client)
-
-    def test_shouldnt_add_two_identical_service_in_cluster(self, sdk_client_fs: ADCMClient):
-        bundle = sdk_client_fs.upload_from_fs(get_data_dir(__file__, 'cluster_simple'))
-        cluster = bundle.cluster_create(utils.random_string())
+                _create_service(cluster, utils.random_string())
+        with allure.step("Check error about valid integer"):
+            assert e.value.error.title == "400 Bad Request"
+            assert e.value.error["prototype_id"][0] == "A valid integer is required."
```

Comment:
This test should be moved to Unit tests too

Reply:
There is no cluster service tests in unit tests. Are you sure we have to move this test?

Reply:
Since you have already started this activity, yes I'm sure.
@grey-kristy Seems like we need your help here with unit tests

Reply:
[done](https://github.com/arenadata/adcm/pull/791/commits/277da469151975789e8f92b7665507ddd0f1b3a2)